packages/core/test/actor.test.ts

```diff
@@ -787,77 +795,6 @@ describe('communicating with spawned actors', () => {
 
     parentService.start();
   });
-
-  it.skip('should be able to name existing actors', (done) => {
```

Comment:
I don't think this test has merit. It doesn't make sense to `spawn` an existing actor ref